src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -65,7 +65,10 @@ public PersistentDataStore() {
         UUID uuid = UUID.fromString((String) entity.getProperty("uuid"));
         String userName = (String) entity.getProperty("username");
         Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
-        User user = new User(uuid, userName, creationTime);
+
+        // TODO: get password from Datastore
+
+        User user = new User(uuid, userName, null, creationTime);
```

Comment:
using null here is mysterious and possibly dangerous (eg, if you ever do password.length or password.substring() etc.  If you really want null as a sentinel value, mark the parameter @Nullable (https://stackoverflow.com/questions/14076296/nullable-annotation-usage) in the User class.  Otherwise, you can keep the three argument constructor in User (for now) in addition to the four argument constructor, and have User.java manage its own default/sentinel value.  This also allows User to know when a password exists and when it doesn't. Here it isn't so important, but consider what would happen if you had added this to a site that already had users, some of whom who did not have passwords -- the User class would need to be backward compatible.

Reply:
On the instructions, it says to place null in there but my next pull request fixes that issue and replaces it with actual passwords.

Reply:
SGTM.